### ISSUE TITLE
Error when compiling

### ISSUE THREAD

**Issue by zhengminlai** [rbock/sqlpp11-connector-mysql#75]
I compiled the code successfully yesterday, but today when I compile the newest version of code on branch master, I get the following error.
```
-- Using /usr/lib64/ccache/c++ (compiler id: GNU)
CMake Error at /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  MySQL_INCLUDE_DIR (missing: MySQL_LIBRARY)
Call Stack (most recent call first):
  /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindMySQL.cmake:50 (find_package_handle_standard_args)
  CMakeLists.txt:40 (find_package)

-- Configuring incomplete, errors occurred!
```
I do not understand why it says I am missing MySQL_LIBRARAY, help.

**Comment by kovdan01** [comment]
@zhengminlai Your problem is related to #72.

Please provide:
- your linux distribution
- path to libmariadb.so/libmysqlclient.so (depending on what you use - MariaDB or MySQL)
- and, actually, do you use MariaDB or MySQL?

If you did not try to build from scratch yet (I mean that maybe you use an old build directory with old CMakeCache.txt) - please try to build the library from scratch and let me know about results.

**Comment by kovdan01** [comment]
@zhengminlai Do you still face the issue? If yes, could you please provide the required environment info (see previous comment)? Currently I am not able to reproduce your problem (tried Ubuntu + MySQL, Arch Linux + MariaDB, Windows + MySQL)

**Comment by zhengminlai** [comment]
Linux Distribution: CentOS 7
I use Mysql and the path to libmysqlclient.so is : `/usr/lib64/mysql/`
I tried to build from scratch and it reported the same problem :(

**Comment by kovdan01** [comment]
@zhengminlai Thanks! Reproduced the issue, submitted a fix: #77. Please check if it works for you and write about results.

**Comment by zhengminlai** [comment]
That works. Thanks.
